### ISSUE TITLE
Bug: generated lexer doesn't compile

### ISSUE THREAD

**Issue by ice1000** [JetBrains/gradle-grammar-kit-plugin#9]
Steps to reproduce:

+ clone https://github.com/covscript/covscript-intellij
+ change version of grammar kit plugin in `build.gradle.kts` from `2017.1.1` to `2018.1`
+ remove this code block:

```kotlin
configure<GrammarKitPluginExtension> {
	jflexRelease = "c779429" // 1.7.0
	grammarKitRelease = "2017.1.2"
}
```

And you'll see compilation error.
I guess it's because the jflex skeleton is broken but I'm not sure.

**Comment by hurricup** [comment]
The problem is the JB patched JFlex still contains original skeleton file. 
Previously, this plugin used other repo with explicit patch for this: https://github.com/hurricup/jflex/commit/41f12fa0e46215d8c197ddb4a9cfdb0e1c8eb092
Suppose something similar should be done in https://github.com/JetBrains/intellij-deps-jflex
As workaround you may add a [skeleton file](https://github.com/hurricup/jflex/blob/idea/jflex/src/main/resources/jflex/idea-flex.skeleton) to your project and specify it explicitly

CC @gregsh

**Comment by ice1000** [comment]
So I prefer waiting for the fix. And I can update the version and remove the config code block. :p

**Comment by hurricup** [comment]
Could you please try `jflexRelease="1.7.0-1"`

**Comment by hurricup** [comment]
I've released `2018.1.1` try it

**Comment by ice1000** [comment]
Works like a charm, thanks!
